repo-index.yml

```diff
@@ -1834,3 +1834,24 @@ plugins:
   name: wildcard_plugin
   updated: 2015-07-02T00:00:00Z
   version: 1.0.0
+- authors:
+    - contact: abaker@pivotal.io
+      homepage: https://github.com/gemfire/tanzu-gemfire-management-cf-plugin
+      name: Pivotal
+  binaries:
+    - checksum: 08de98c38fff2ad0c5fde51f37ad9c1b
+      platform: osx
+      url: https://github.com/gemfire/tanzu-gemfire-management-cf-plugin/releases/download/v1.0.1/gemfire-osx
```

Comment:
does the name of the executable have to the the same as the name of the "plugin"?